plugin/vcs/gitlab/gitlab.go

```diff
@@ -416,6 +423,61 @@ func (p *Provider) FetchCommitByID(ctx context.Context, oauthCtx common.OauthCon
 	}, nil
 }
 
+// GetDiffFileList gets the diff files list between two commits.
+func (p *Provider) GetDiffFileList(ctx context.Context, oauthCtx common.OauthContext, instanceURL, repositoryID, beforeCommit, afterCommit string) ([]vcs.FileDiff, error) {
```

Comment:
Could you add a unit test for this method?

Reply:
This is mainly about calling the GitLab API. There's an integration test mock for it https://github.com/bytebase/bytebase/blob/19d4ffcd56716fee89e8945e8ec8f38a9cd52a50/tests/fake/gitlab.go#L87

Reply:
We have tons of examples in https://github.com/bytebase/bytebase/blob/main/plugin/vcs/gitlab/gitlab_test.go. I've caught non-zero bugs in our code by writing those unit tests that are _supposed_ to be caught by our integration tests.

Reply:
Thanks! I'll add a ut for `GetDiffFileList`.